resources/views/groups/create.blade.php

```diff
@@ -2,76 +2,70 @@
-                                        <strong>{{ $message }}</strong>
-                                    </span>
-                                @enderror
-                            </div>
-                        </div>
+                @error('name')
+                <span class="invalid-feedback" role="alert">
+                  <strong>{{ $message }}</strong>
+                </span>
+                @enderror
+              </div>
+            </div>
 
-                        <div class="form-group row">
-                            <label for="password" class="col-md-4 col-form-label text-md-right">{{ __('Password') }}</label>
+            <div class="form-group row">
+              <label for="information" class="col-md-4 col-form-label text-md-right">{{ __('Group Information') }}</label>
 
-                            <div class="col-md-6">
-                                <input id="password" type="password" class="form-control @error('password') is-invalid @enderror" name="password" required autocomplete="new-password">
+              <div class="col-md-6">
+                <input id="information" type="textarea" class="form-control @error('information') is-invalid @enderror" name="information" value="{{ old('information') }}">
 
-                                @error('password')
-                                    <span class="invalid-feedback" role="alert">
-                                        <strong>{{ $message }}</strong>
-                                    </span>
-                                @enderror
-                            </div>
-                        </div>
+                @error('information')
+                <span class="invalid-feedback" role="alert">
+                  <strong>{{ $message }}</strong>
+                </span>
+                @enderror
+              </div>
+            </div>
 
-                        <div class="form-group row">
-                            <label for="password-confirm" class="col-md-4 col-form-label text-md-right">{{ __('Confirm Password') }}</label>
+            <div class="form-group row">
+              <label for="categories" class="col-md-4 col-form-label text-md-right">{{ __('Select the Group Category') }}</label>
 
-                            <div class="col-md-6">
-                                <input id="password-confirm" type="password" class="form-control" name="password_confirmation" required autocomplete="new-password">
-                            </div>
-                        </div>
+              <div class="col-md-6">
+                <select name="categories" id="categories" value="">
+                  <?php
+                  foreach($categories as $category)
+                  {
+                    print('<option id="category" type value="'.$category->id.'">'.$category->name.'</option>'); 
+                  }
+                  ?>
+                </select>
+            </div>
```

Comment:
Laravel のbladeでは <?php  printなどは基本的に使用しないです！
```suggestion
                <select name="category_id" id="categories" value="">
                  @foreach($categories as $category)
                    <option id="category" value="{{ $category->id }}">{{ $category->name }}</option>
                  @endforeach
                </select>
```